pkgdb/src/pkgdb/input.cc

```diff
@@ -126,7 +126,8 @@ PkgDbInput::closeDbReadWrite()
 
 
 /* -------------------------------------------------------------------------- */
-
+// NOLINTBEGIN(readability-function-cognitive-complexity)
+// TODO: reduce complexity of this function
```

Comment:
What's the precedent of this kind of thing?  I have cycles I could devote to it now, but only b/c I haven't selected already prioritized work in the backlog. Or do we add a ticket for later prioritization.  Probably the later given that user facing changes would be more productive for the hard release?  Are 'tech-debt' tickets tolerated? :)

Reply:
I'm not sure if these are on the radar just now, I add todos to these obvious big bois because for that reason. Also I don't quite have the domain knowledge to "just do them" while working on the warnings.

Reply:
Nm, I'll look to address this as part of issue #1037 .